urdf/include/urdf/model.h

```diff
@@ -48,9 +49,18 @@
 namespace urdf
 {
 
+// Forward Declaration
```

Comment:
Nit: may want to say that this is a forward declaration for the purposes of PIMPL.  It's just a bit more information for the reader.

Reply:
Documented forward declaration is for PIMPL in d079336